tox.ini

```diff
@@ -337,8 +337,8 @@ deps =
     framework_sanic-sanic2109: sanic<21.10
     framework_sanic-sanic2112: sanic<21.13
     framework_sanic-sanic2203: sanic<22.4
-    ; Temporarily pin this to the second to last release
-    framework_sanic-saniclatest: sanic==22.6.2
+    framework_sanic-sanic2290: sanic<22.9.1
```

Comment:
Can we change this to specifically test the latest version of sanic so we're alerted when a new release has breaking changes?

Reply:
I think this is already doing that.
```
framework_sanic-sanic2290: sanic<22.9.1
framework_sanic-saniclatest: sanic
```

Reply:
Ah yes, all good then!